release-scripts/build-non-x86-tarballs.sh

```diff
@@ -20,7 +27,7 @@ rm cmdstan-${VERSION}/bin/*-stanc
 
 for ARCH_NAME in ${ARCHS[@]}
 do
-    mv "linux-${ARCH_NAME}-stanc" "cmdstan-${VERSION}/bin/linux-${ARCH_NAME}-stanc"
-    tar -cvf "cmdstan-${VERSION}-linux-${ARCH_NAME}.tar.gz" "cmdstan-${VERSION}"
-    rm "cmdstan-${VERSION}/bin/linux-${ARCH_NAME}-stanc"
+    mv "linux-${ARCH_NAME}-stanc" "cmdstan-${VERSION}/bin/linux-stanc"
```

Comment:
If the name is linux-arm64-stanc, cmdstan starts to download the binary, because it looks for linux-stanc.